hikari/impl/entity_factory.py

```diff
@@ -1913,6 +1916,13 @@ def serialize_command_option(self, option: commands.CommandOption) -> data_bindi
         if option.options is not None:
             payload["options"] = [self.serialize_command_option(suboption) for suboption in option.options]
 
+        # because discord does not allow floating point min and max values with integer options we have to round
```

Comment:
Still dont believe this is the way to go. We should just be adding it raw, since if not it could confuse some people.

An example of a case where this can be confusing is when a user specifies `INTEGER` by mistake and then starts receiving values under (lets say) `1.2` . This could take some time to debug, instead of just erroring as soon as the command is created, since (and I cant really test this until the weekend) Discord should error